source/game/g_svcmds.cpp

```diff
@@ -429,6 +583,8 @@ void G_RemoveCommands( void )
 	trap_Cmd_RemoveCommand( "removeip" );
 	trap_Cmd_RemoveCommand( "listip" );
 	trap_Cmd_RemoveCommand( "writeip" );
+	trap_Cmd_RemoveCommand( "ban" );
+	trap_Cmd_RemoveCommand( "removeban" );
 
```

Comment:
You should probably also remove `mute` and `removemute`

Reply:
happens automatically with cgcmds